automation/DeployToHPVS.1

```diff
@@ -0,0 +1,74 @@
+.\" 
+.\" Copyright (C) 2021 IBM. All Rights Reserved.
+.\" 
+.\" See the LICENSE file in the root directory
+.\" of this source tree for licensing information.
+.\"
+.if \n[.g] .mso www.tmac
+.TH DeployToHPVS.sh 1 2021-01-25 Linux "DeployToHPVS.sh man page"
+.SH NAME
+DeployToHPVS.sh \- Sign an IBM Fully Homomorphic Encryption Toolkit container image and deploy it to a new Hyper Protect Virtual Server instance in the IBM public cloud.
+.SH SYNOPSIS
+\fBDeployToHPVS.sh\fR [OPTION...] [\fIcontainerOS\fR] 
+.SH DESCRIPTION
+Automate the process of signing an IBM Fully Homomorphic Encryption (FHE) Toolkit virtual server/client image and deploying it to a new Hyper Protect Virtual Server (HPVS) instance in the IBM public cloud.
+.SH OPTIONS
+.TP
+.BR \-c " " \fIconfigFile\fR
+Generate a new configuration file at the given path using an interactive wizard.  The new configuration file with the name and path specified.  Mutually exclusive with -f.
+.TP
+.BR \-f " " \fIconfigFile\fR
+ Path to the configuration file for this script.  If left unspecified, the script will use the default file DeployToHPVS.conf, located in the same directory as the script.  Mutually exclusive with -c.
+.TP
+.BR \-h
+ Display help information.
+.TP
+.BR \-l
+ Deploy a locally-built FHE Toolkit container image generated by BuildDockerImage.sh.  If left unspecified, this script will assume that you are deploying a pre-built toolkit fetched from from
+.URL "https://hub.docker.com/u/ibmcom" "ibmcom"
+on
+.URL "https://hub.docker.com/" "DockerHub".
+.TP
+.IR containerOS
+Indicates the operating system to be deployed into the FHE Toolkit container.  If left unspecified, defaults to \fBfedora\fR.
+.PP
+.RS
+Valid values are as follows:
+.PP
+.RS
+\fBalpine\fR
+.PP
+\fBfedora\fR
+.PP
+\fBubuntu\fR
+.RE
+.fi
+.SH NOTES
+.TP
+Only Alpine, Fedora, and Ubuntu are supported as container operating systems as those are the only ones that the FHE project has built for s390x.
+.TP
+The default behavior of this script is to attempt to use the ibmcom pre-built toolkit available from Docker Hub.  This behavior can be overridden with the \fB-l\fR option flag.
+.TP
+By default, all commands executed by these functions are silenced except for when a nonzero RC is returned.  To display the commands being executed and their output, issue:
+.PP
+.RS
+\fBset\fR DEPLOY_TO_HPVS_DEBUG
+.RE
+.fi
+.SH EXAMPLES
+To download an s390x Fedora-based FHE container from DockerHub and deploy it to HPVS using a custom configuration file named 'DeployToHPVS.conf.mine':
+.PP
+.nf
+.RS
+docker pull ibmcom/fhe-toolkit-fedora-s390x
+DeployToHPVS.sh -f DeployToHPVS.conf.mine fedora
+.RE
+.fi
+.PP
+.SH BUGS
+No known bugs.
+.SH AUTHOR
+.PP
+Lei A.B. Wang (wlwangwl@cn.ibm.com)
+.PP
+Dan FitzGerald (danfitz@us.ibm.com)
```

Comment:
I think we need to add someone as the maintainer here. Also you and Dean edited this so you should be authors. I would tag yourself @boland25 as maintainer as well.

Reply:
ok, will add